mars/deploy/oscar/session.py

```diff
@@ -78,19 +81,21 @@ async def init(cls,
             from .local import new_cluster
             return (await new_cluster(address, **kwargs)).session
 
-        session_api = await SessionAPI.create(address)
+        web_address = kwargs.pop('web_address', None)
```

Comment:
I don't think it a nice design to have a separate ``web_address`` argument. Simply parse ``address`` and check the protocol part instead.